Cargo test error in td-shim/src/e820.rs

```
   Compiling td-shim v0.1.0 (/home/liuwei/td-shim/td-shim)
error[E0425]: cannot find value `MAX_E820_ENTRY` in this scope
  --> td-shim/src/e820.rs:64:35
   |
61 |     fn test_e820_entry_size() {
   |                            - help: you might be missing a type parameter: `<MAX_E820_ENTRY>`
...
64 |             size_of::<[E820Entry; MAX_E820_ENTRY]>(),
   |                                   ^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `MAX_E820_ENTRY` in this scope
  --> td-shim/src/e820.rs:65:18
   |
65 |             20 * MAX_E820_ENTRY
   |                  ^^^^^^^^^^^^^^ not found in this scope
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -57,6 +57,9 @@ jobs:
         with:
           command: install
           args: cargo-xbuild
+      
+      - name: Test Shim Crates
+        run: make test 
 
       - name: Build TdShim
         uses: actions-rs/cargo@v1
@@ -110,6 +113,7 @@ jobs:
       - name: Install QEMU (Linux)
         run: sudo apt update && sudo apt install qemu-system-x86
         if: runner.os == 'Linux'
+
       - name: Install QEMU (Windows)
         run: |
           choco install qemu --version 2021.5.5

--- a/Makefile
+++ b/Makefile
@@ -115,7 +115,7 @@ clean-%:
 	${CARGO} +${STABLE_TOOLCHAIN} clean -p $(patsubst clean-%,%,$@) ${BUILD_TYPE_FLAG}
 
 test-%:
-	${CARGO} +${STABLE_TOOLCHAIN} test -p $(patsubst test-%,%,$@) ${BUILD_TYPE_FLAG}
+	${CARGO} +${NIGHTLY_TOOLCHAIN} test -p $(patsubst test-%,%,$@) ${BUILD_TYPE_FLAG}
 
 # Targets for normal library/binary crates
 nightly-build-%:

--- a/td-layout/src/metadata.rs
+++ b/td-layout/src/metadata.rs
@@ -421,7 +421,7 @@ mod tests {
         #[cfg(not(feature = "boot-kernel"))]
         assert_eq!(size_of::<TdxMetadata>(), 224);
         #[cfg(feature = "boot-kernel")]
-        assert_eq!(size_of::<TdxMetadata>(), 256);
+        assert_eq!(size_of::<TdxMetadata>(), 288);
     }
 
     #[test]

--- a/td-shim-tools/src/enroller.rs
+++ b/td-shim-tools/src/enroller.rs
@@ -261,13 +261,15 @@ pub fn create_key_file(key_file: &str, hash_alg: &str) -> io::Result<FirmwareRaw
 
 #[cfg(test)]
 mod test {
+    use std::str::FromStr;
+
     use super::*;
     use td_uefi_pi::pi::guid;
 
     #[test]
     fn test_firmware_file() {
         // {214D240F-77A3-441B-9DA8-C588E43192C1}
-        let name = guid::Guid::parse_str("214D240F-77A3-441B-9DA8-C588E43192C1").unwrap();
+        let name = guid::Guid::from_str("214D240F-77A3-441B-9DA8-C588E43192C1").unwrap();
         let size: usize = size_of::<FvFfsFileHeader>();
 
         let mut ff = FirmwareRawFile::new(name.as_bytes());

--- a/td-shim/src/e820.rs
+++ b/td-shim/src/e820.rs
@@ -56,6 +56,7 @@ impl E820Entry {
 mod tests {
     use super::*;
     use core::mem::size_of;
+    const MAX_E820_ENTRY: usize = 128;
 
     #[test]
     fn test_e820_entry_size() {

